api_end_end_testing/src/main/java/tds/user/api/model/RoleAssociation.java

```diff
@@ -0,0 +1,42 @@
+package tds.user.api.model;
```

Comment:
I do not think the class needs to be in a test package.  That being said I would recommend that we move it into `test/java` rather than `main/java`.  The reason I say this is because the code in `main/java` doesn't really serve any purpose outside of running the integration tests.

If this code was executed outside of maven (like an executable jar) then I'd say it should be in main.